_src/_assets/css/_globals.scss

```diff
@@ -36,4 +46,24 @@ img, svg {
 
 body {
 	font-family: $body;
-}
\ No newline at end of file
+  line-height: 1.4;
+  color: $text-color-dark;
+  background-color: #fff;
+}
+
+// Content between header and footer
+.body {
+  font-size: $size-3;
+}
+
+a {
+  color: $link-color;
+
+  &:hover {
+    color: darken($link-color, 10%);
+  }
+}
+
+.text-left {
+	text-align: left;
```

Comment:
okay so this is literally the worst sentence i have ever typed into a github comment, and for that i am truly, incredibly sorry

_okay so with that said_

Would you be willing to revisit the whitespace in this PR for consistent indents? We _literally_ just added an .editorconfig in #48, so that’s a bit on us; but if you’d be willing to make this consistent, we’d v v deeply appreciate it. Thank you!